app/components/activities/member_activity_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= t(
+  @activity[:key],
```

Comment:
**nitpick:** You should have tests / previews for all these components

Reply:
Added in 5ca4ad87cf0bde4d121092a8bd7a52fd69062f77 & 9d09924142c39be63e86d3a213ca6ebc90f15d27. I only added a component preview for the activity component as the other components under `components/actiivities/`
only return the translated strings that the activity component displays